### PR TITLE
Initial com.mousepawmedia.omission Flatpak manifest

### PR DESCRIPTION
**Pull request by CodeMouse92**
As far as I can tell, this is the first Kivy-based application to be packaged in Flatpak. Thus, I'd appreciate any suggestions on how to improve this.

Be aware, omitting some libraries will still allow this to build, but will lead to runtime errors, due to how Kivy works. It's not ideal, but hey.

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/mousepawmedia/omission
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by CodeMouse92** [comment]
bot, build com.mousepawmedia.omission

File: com.mousepawmedia.omission.yaml [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+      buildsystem: autotools
+      sources:
+        - type: archive
+          url: https://github.com/astrand/xclip/archive/0.13.tar.gz
+          sha256: ca5b8804e3c910a66423a882d79bf3c9450b875ac8528791fb60ec9de667f758
+
+    - name: mtdev
+      buildsystem: autotools
+      sources:
+        - type: archive
```
Comment (nedrichards) [inline com.mousepawmedia.omission.yaml:125]: to fix the aarch64 build here try and add something like https://github.com/flathub/com.tux4kids.tuxmath/blob/adc11b752a572c744b2041ba43d65545af121207/com.tux4kids.tuxmath.json#L51

**Comment by CodeMouse92** [comment]
bot, build com.mousepawmedia.omission

File: com.mousepawmedia.omission.yaml (hunk 1 above)
Comment (CodeMouse92) [reply com.mousepawmedia.omission.yaml:125]: It's really painfully not working, even still. I'm actively considering commenting that module out until somebody complains about it. Would you believe I've been trying to package this thing for FIVE years?

**Comment by CodeMouse92** [comment]
bot, build com.mousepawmedia.omission

**Comment by CodeMouse92** [comment]
Crap. I give up. I'm going to disable aarch64.

**Comment by CodeMouse92** [comment]
bot, build com.mousepawmedia.omission

File: com.mousepawmedia.omission.yaml (hunk 1 above)
Comment (nedrichards) [reply com.mousepawmedia.omission.yaml:125]: No! Wow, that is persistence. I can try and take a closer look a little later.

Comment (CodeMouse92) [reply com.mousepawmedia.omission.yaml:125]: Nah, I just dropped aarch64 support. I'd rather get *this* published and address aarch64 later. I just want this shipped.

**Comment by CodeMouse92** [comment]
@nedrichards Just to be clear, I *am* ready for review on this. I only decided just to drop aarch64 support for the time being, as Intel/AMD is the primary target anyway. Hoping to get this landed soon. :) Maybe we can fix aarch64 later, after this is live for everyone else.

**Comment by nedrichards** [comment]
/merge

There's some chatter about issue with /dev/input/event14 and /event17 but that doesn't seem to affect the gameplay and further isn't actually solved by unblocking access to more of dev with --device=all so I think this is good to go. As you say, we can have another look at aarch64 in a PR.

**Pull request closed without merging**